config/wmflabs-staging/97-staging.yaml

```diff
@@ -0,0 +1,2 @@
+wsgi:
+  host: labels-staging.wmflabs.org
```

Comment:
I think this should be handled via puppet instead of here. That's fairly easy. I do it today.